src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/ISyntaxFactsExtensions.cs

```diff
@@ -281,6 +281,9 @@ public static bool IsTrueLiteralExpression(this ISyntaxFacts syntaxFacts, [NotNu
         public static bool IsAwaitExpression(this ISyntaxFacts syntaxFacts, [NotNullWhen(true)] SyntaxNode? node)
             => node?.RawKind == syntaxFacts.SyntaxKinds.AwaitExpression;
 
+        public static bool IsTargetTypedNewExpression(this ISyntaxFacts syntaxFacts, [NotNullWhen(true)] SyntaxNode? node)
+            => syntaxFacts.IsTargetTypedNew(node);
```

Comment:
perhaps consider a different naming to not conflict with target-typed `new[]`

Reply:
Done. Thanks